tutorials/scripting/gdscript/gdscript_exports.rst

```diff
@@ -26,7 +26,7 @@ them visible and editable in the editor. This way, artists and game designers
 can modify values that later influence how the program runs. For this, a
 special export syntax is provided.
 
-Exporting can only be done with built-in types or objects derived from the :ref:`Resource class <class_Resource>`.
+For more information about export annotations see :ref:`GDScript annotations <class_@GDScript>`.
```

Comment:
I agree that the original sentence is no longer valid, but that's a weird thing to say in an article that exists to provide more information about export annotations itself :)